src/app/components/routing/application-routing.component.ts

```diff
@@ -38,9 +38,8 @@ export class ApplicationRoutingComponent implements OnInit {
       if (bookingFeatureToggle && AppUtils.isBookableAndJudicialRole(userDetails)) {
         return this.router.navigate([ApplicationRoutingComponent.bookingUrl]);
       }
-      if (userDetails && userDetails.userInfo && userDetails.userInfo.roles
-        && !userDetails.userInfo.roles.includes('pui-case-manager')) {
-        const userRoles = userDetails.userInfo.roles;
+      if (userDetails?.userInfo?.roles && userDetails.userInfo.roles.includes('pui-case-manager')) {
```

Comment:
Please add checks to the other part of the if statement: userDetails?.userInfo?.roles?.includes

Reply:
And remove the &&